### PR TITLE
net-misc/freerdp: bump to 1.2.1_pre20160606

### PR DESCRIPTION
**Pull request by diogocp**
Newer versions of net-misc/remmina need an updated snapshot of FreeRDP (see [comment in bug 571026](https://bugs.gentoo.org/show_bug.cgi?id=571026#c2)). I picked the same one that [Arch is using](https://www.archlinux.org/packages/community/x86_64/freerdp/), so it should be fairly stable.

Only a few trivial changes in the ebuild:
  1. updated copyright year
  2. updated commit hash
  3. dropped all keywords from arch to ~arch
  4. removed PATCHES (all merged upstream)

CC @floppym

### CODE REVIEW

File: net-misc/freerdp/freerdp-1.2.1_pre20160606.ebuild [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="5"
```
Comment (SoapGentoo) [inline net-misc/freerdp/freerdp-1.2.1_pre20160606.ebuild]: why no `EAPI=6`?

**Comment by diogocp** [comment]
I have updated the ebuild according to @SoapGentoo's comments.

File: net-misc/freerdp/freerdp-1.2.1_pre20160606.ebuild [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+		app-text/docbook-xml-dtd:4.1.2
+		app-text/xmlto
+	) ) )
+"
+
+DOCS=( README )
+
+src_configure() {
+	local mycmakeargs=(
+		$(cmake-utils_use_with alsa ALSA)
```
Comment (SoapGentoo) [inline net-misc/freerdp/freerdp-1.2.1_pre20160606.ebuild]: Are you sure these work in `EAPI=6`? Have you tried the ebuild?

Comment (diogocp) [inline net-misc/freerdp/freerdp-1.2.1_pre20160606.ebuild]: No, I just trusted you ;)

Fixed.

**Comment by floppym** [comment]
Thanks for the pull request, but I'm planning to make some additional changes before I add another snapshot. It should be coming soon.

**Pull request closed without merging**